TR-050 Incremento

JoaoPauloRocon opened the pull request:
Feito a implementação do parâmetro  "aberta" na requisição para listagem de feiras

JoaoPauloRocon committed:
TR-050 Incremento

Feito a implementação do parâmetro  "aberta" na requisição para listagem de feiras

Prezotti reviewed (changes requested):
O controller está ok porém no service utilize os métodos do JPA para já buscar a feira aberta.
Ex.: findAllByAbertaTrue()

JoaoPauloRocon committed:
TR-050

Feito a implementação do parâmetro "aberta" na requisição para listagem de feiras

Prezotti reviewed (changes requested):
Retirar o método antigo

JoaoPauloRocon committed:
TR-050 Incremento

Feito a implementação do parâmetro "aberta" na requisição para listagem de feiras

Prezotti reviewed (approved):
Aprovado!

Prezotti reviewed (approved):
Aprovado!
